src/Model/ApplicationToken.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/*
+ * This file is part of the Secret Santa project.
+ *
+ * (c) JoliCode <coucou@jolicode.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace JoliCode\SecretSanta\Model;
+
+class ApplicationToken
+{
+    /**
+     * @param string               $token
+     * @param array<string, mixed> $values
+     */
+    public function __construct(
+        private string $token,
+        private array $values = [],
```

Comment:
Not a big fan of this name `values`. What about `context`?

Reply:
Fine by me